README.md

```diff
@@ -42,18 +42,26 @@ objects like permissions, custom resources and corresponding StatefulSets.
 > For install instructions on Openshift, head to the
 > [official help page](https://www.dynatrace.com/support/help/shortlink/full-stack-dto-k8)
 
-To create the namespace and apply the operator run the following commands
+First, create the namespace
```

Comment:
Remove these instructions completely and link to the official documentation. these are outdated and we should not have them in multiple places.

Reply:
Removed instructions and just kept reference to official docu.